bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/gateway.xml

```diff
@@ -69,7 +69,7 @@
 		<category>Moisture</category>
 		<tags>
 			<tag>Measurement</tag>
-			<tag>Moisture</tag>
+			<tag>Rain</tag>
```

Comment:
Rain != Moisture

I think the tag should be changed or removed.

Reply:
The choices are Rain Humidity or nothing. Your choice.

Reply:
I'd vote for removing it then.

Reply:
I read the ReadMe and it seems that the channel relates to soil wetness. As we have no specific property for that, I think Humidity is the closest.

Reply:
Tags should be precise in some way. Humidity is strictly about air. Moisture indicates the amount of liquid present in any material. While it is close, it is not the same and therefore i opt to either delete this tag, or create a core PR to add moisture (also not sure if that should be a percentage (like humidity) or ml per mg).